src/cluster_legacy.h

```diff
@@ -352,6 +352,7 @@ struct _clusterNode {
     mstime_t fail_time;                     /* Unix time when FAIL flag was set */
     mstime_t repl_offset_time;              /* Unix time we received offset for this node */
     mstime_t orphaned_time;                 /* Starting time of orphaned primary condition */
+    mstime_t last_connect_attempted;        /* Unix time we last tried to establish a link */
```

Comment:
Better to term it as 

```suggestion
    mstime_t outbound_link_attempt_time; /* Unix time we last tried to establish an outgoing link */
```